CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+# Contributing
+
+The following documentation is only for the maintainers of this repository.
+
+- [Monorepo setup](#monorepo-setup)
+- [Installation](#installation)
+- [Develop the CLI packages](#Develop-the-CLI-packages)
+- [Release the packages](#release-the-packages)
+- [Available commands](#commands)
+- [CI](#ci)
+- [Add a new package to the monorepo](#add-a-new-package-to-the-monorepo)
+
+## Monorepo setup
+
+This repository is managed as a monorepo that is composed of many npm packages.
+
+For more information on monorepo:
+
+- [Babel GitHub](https://github.com/babel/babel/blob/master/doc/design/monorepo.md)
+- [Shopify GitHub](https://github.com/Shopify/quilt/blob/master/Decision%20records/00%20-%20Use%20a%20Lerna%20monorepo.md)
+- [Google](https://www.google.com/search?q=monorepo)
+
+### PNPM workspace
+
+This monorepo is using PNPM workspace feature to handle the installation of the npm dependencies and manage the packages interdependencies.
+
+It's important to note that PNPM workspace will **hoist** the npm dependencies at the root of the workspace. This means that there might not be a *node_modules* directory nested in the packages directories. The npm dependencies are installed in a *node_modules* directory at the root of the workspace and a single *pnpm-lock.yaml* file is generated at the root of the workspace.
+
+## Installation
+
+This project uses PNPM workspace. Therefore, you must [install PNPM](https://pnpm.io/installation):
+
+To install the project, open a terminal at the root of the workspace and execute the following command:
+
+```bash
+pnpm i
+```
+
+## Develop the CLI packages
+
+The following documentation is a brief overview of the tools and processes involved in the development of the CLI packages.
+
+> To develop for each package, you can run `pnpm dev` from the project root folder. This will build all the content of the `packages` folder.
```

Comment:
I would rephrase for something like: "To start developing, open a terminal at the root of the project and execute the `pnpm dev` command. This will build all the packages and start them in "watch" mode."

Reply:
By the way, isn't the `build:dev` script missing a `--watch` option?

Reply:
After reading the "Working in `@workleap/create-project`" and "Working in `@workleap/foundry`" sections, it's not clear to me if you recommend starting both projects in dev mode from the root and if it's better to start both of them individually.

Do you have an opinion about this?

Reply:
I think that starting `dev` from the root can be useful in situations where you need to have both running at the same time, like "adding a template" or "adding an option".

But in other situations, it's not useful.

My personal opinion on that is if I need to run both, I will simply start them on two terminal sessions, and that's all.

Reply:
Got it! Make sense. I think it would be valuable to add what you described in the docs so the contributors are aware of this distinction!